docs/monitoring/metrics.md

```diff
@@ -563,6 +563,7 @@ reporters will be instantiated on each job and task manager when they are starte
 - `metrics.reporter.<name>.factory.class`: The reporter factory class to use for the reporter named `<name>`.
 - `metrics.reporter.<name>.interval`: The reporter interval to use for the reporter named `<name>`.
 - `metrics.reporter.<name>.scope.delimiter`: The delimiter to use for the identifier (default value use `metrics.scope.delimiter`) for the reporter named `<name>`.
+- `metrics.reporter.<name>.scope.variables.excludes`: (optional) A semi-colon (;) separate list of variables that should be ignored by tag-based reporters. 
```

Comment:
How about `metrics.reporter.<name>.scope.variables.excluded`?
Moreover, I think we might give some explanations to so called tag-base reporters or give some examples.

Reply:
Not a fain of `excluded`, but that may very well be since I'm used to `excludes` from maven. I'll add an example.